src/map-performance.spec.js

```diff
@@ -15,15 +18,25 @@ const cwd = process.cwd()
 const {log: _log} = console
 const log = _log.bind(console)
```

Comment:
FWIW I don't think this is necessary anymore - `const {log} = console` should be fine...?